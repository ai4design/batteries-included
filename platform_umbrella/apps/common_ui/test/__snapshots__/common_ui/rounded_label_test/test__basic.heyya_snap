<div class="text-xs font-bold leading-sm uppercase rounded-lg border shadow-md inline-flex items-center m-1 px-4 py-2">
  Basic pill
</div>