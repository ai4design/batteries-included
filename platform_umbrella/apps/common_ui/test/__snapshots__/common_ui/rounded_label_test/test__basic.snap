"<div class=\"text-xs font-bold leading-sm uppercase rounded-full border inline-flex items-center m-1 px-4 py-1\">\n  Basic pill\n</div>"